pykob/config.py

```diff
@@ -241,4 +245,14 @@ def readConfig():
 
 # ### Mainline
 readConfig()
+
+PortOverride = argparse.ArgumentParser(add_help=False)
+PortOverride.add_argument("-p", "--port", default=Port, help="The  name of the serial port to use ", metavar="portname", dest="Port")
+
+SoundtOverride = argparse.ArgumentParser(add_help=False)
+SoundtOverride.add_argument("-a", "--sound", default='ON' if Sound else 'OFF', choices=['ON', 'OFF'], help="'ON' or 'OFF' to indicate whether morse audio should be generated", metavar="sound", dest="Sound")
```

Comment:
So `Sound` can't be kept as a boolean? That seems like a problem. I don't like each application having to do 'ON'|'OFF' processing. Also, does `saveConfig` still work if an application wants to update the configuration?

Reply:
Well, I'm fine with the argument parameter coming out as a string and this just has the user interpret args.Sound as a string.  I see config.py has setSound do a "_sound = strtobool(str(sound))" which converts 'OFF' to 'False' (instead of casting it with bool(str) which interprets 'OFF' as 'True'.  Are you suggesting I just use "strtobool" above instead?

Or do you want to be able to specify "-a ON" and "-a OFF" but have args.Sound come out as a boolean?  Is there an easy way to do that in the arg. parser?

Note that one alternative would be to allow (mutually exclusive) overrides '-a' to mean "audio ON" and, say, "-q" (quiet) for "audio OFF" and use action=store_true/store_false but I think I prefer the "-a ON" / "-a OFF" style.

Reply:
I would use `strtobool` as that handles 'YES', 'TRUE', 'ON', '1' for `True` and 'NO', 'FALSE', 'OFF', '0' for `False`. So they could enter '--sound on', '--sound yes', '-a off', etc.